windows/src/developer/TIKE/xml/layoutbuilder/physical-keyboard-template.js

```diff
@@ -0,0 +1,83 @@
+  "row": [
```

Comment:
What does this file correspond to?  It doesn't have both the left and right versions of the chiral modifiers, so I'm curious.

Reply:
This is a template of the base physical layout used during import from a KVK file to a touch layout. The modifiers that are used are loaded as longpress options under the LControl key. See `TframeTouchLayoutBuilder.SetupModifierKeysForImportedLayout`